.github/semantic.yml

```diff
@@ -0,0 +1,4 @@
+titleOnly: false
```

Comment:
If we want to use squash marges here this should match the one in DV https://github.com/dhis2/data-visualizer-app/blob/master/.github/semantic.yml

Reply:
Updated